src/dlgConnectionProfiles.cpp

```diff
@@ -817,6 +821,9 @@ void dlgConnectionProfiles::slot_item_clicked(QListWidgetItem* pItem)
         if (profile_name == QStringLiteral("Reinos de Leyenda")) {
             host_url = QStringLiteral("reinosdeleyenda.es");
         }
+        if (profile_name == QStringLiteral("Test_Profile")) {
```

Comment:
What about `Mudlet self-test` - it's a bit lengthy, but clearly communicates the purpose. Otherwise people can think it's a profile they can use for testing their things and so on.